src/gsi_ncdiag/gsi_ncdiag.py

```diff
@@ -1765,18 +1792,34 @@ def toGeovals(self, OutDir, clobber=True):
             elif vname in geovals_vars.keys():
                 if (len(var.dimensions) == 1):
                     dims = ("nlocs",)
+                    dim_names = ["Location"]
                 elif "_levels" in vname:
                     dims = ("nlocs", "nlevsp1")
+                    dim_names = ["Location", "Layer"]
                 else:
                     dims = ("nlocs", "nlevs")
+                    dim_names = ["Location", "Level"]
                 var_out = ncout.createVariable(geovals_vars[vname], var.dtype, dims)
                 vdata = var[...]
                 var_out[...] = vdata
+                if (TotalBias):
+                    # compute final bias correction
+                    key1 = 'Obs_Minus_Forecast_adjusted'
+                    key2 = 'Obs_Minus_Forecast_unadjusted'
+                    if key1 in self.df.variables.keys() and key2 in self.df.variables.keys():
+                        tmp = self.var(key1) - self.var(key2)
+                        iodavar = 'GsiBc'
+                        gvname = vname, iodavar
+                        outdata[gvname] = np.reshape(tmp, dim)
+                        VarDims[gvname] = dim_names
+                    else:
+                        print(f' ... can not add total bias missing {key1} or {key2} from input file')
```

Comment:
@danholdaway and @CoryMartin-NOAA wasn't having luck running Ozone with ncDiag:

```
Traceback (most recent call last):
  File "/work2/noaa/jcsda/bruston/work/JCSDA/JEDI/ioda-bundle-gnu/iodaconv/src/gsi_ncdiag/gsi_ncdiag.py", line 1862, in toIODAobs
    tmp = self.var('Input_Observation_Error')
```

Reply:
I don't think we've ever tried the operations diags for ozone through the converter, it's possible there is a slightly different variable name that needs to be changed.